libraries/RTC/examples/HibernateRTCAlarm/HibernateRTCAlarm.ino

```diff
@@ -0,0 +1,62 @@
+/*
```

Comment:
Let me explain this another way, this example has been deleted from relase version of XMC-for-Arduino something like 3 years ago.

The person who wrote this example did not understand how Hibernate works, when the CPU comes out of hibernate the CPU is RESET apart from hibernate falags. This means the code will run Setup and Loop again

The function alarmMatch() will NEVER be reached.

xmc.enableHibernate was removed from DeviceControl library ages ago this example should be removed.

Reply:
This suggests your base for develop-2.x was based on a VERY OLD version of the package

Reply:
> A better rework would be to move the xmc_pi_conf and xmc_i2c_conf files to variants folder, for easier maintainability. Note this makes NO difference to compilation paths if in same folder as pins_arduino.h

Yes, that will be considering in an upcoming improvement 👍